core-bundle/src/Resources/contao/templates/_new/content_element/toplink.html.twig

```diff
@@ -0,0 +1,36 @@
+{% trans_default_domain "contao_default" %}
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{% block content %}
+    {% block link %}
+        {% set link_attributes = attrs(link_attributes|default)
+            .set('href', '#')
```

Comment:
```suggestion
            .set('href', '#top')
```

And use the correct href here for the base-tag-less future ☺️

Reply:
see 006c4ab3d3f1e70c07d2c1bf91f8e11a3a0af798